### PR TITLE
RDM-11396 - data-store wiring and cleanup

### PR DESCRIPTION
**Pull request by smathangi**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by sullivangareth — APPROVED** [review]

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -67,36 +67,23 @@ azure:
...
-ccd:
-  case-definition:
-    host: ${DEFINITION_STORE_HOST:http://localhost:4451}
 
 documentTTL: ${DOCUMENT_TTL:3600000}
 
 documentStoreUrl: ${DM_STORE_BASE_URL:http://localhost:4506}
 caseDataStoreUrl: ${CCD_DATA_STORE_API_BASE_URL:http://localhost:4452}
-swaggerUrl: ${CASE_DOCUMENT_SWAGGER_AM_URL:localhost:4455}
-caseDocumentAmUrl: ${TEST_URL:http://localhost:4455}
```
Comment (mattnayler) [inline src/main/resources/application.yaml:99]: You may wish to update https://github.com/hmcts/ccd-docker/blob/master/compose/case-document-am.yml to match

**Review by mattnayler — APPROVED** [review]

File: src/main/resources/application.yaml (hunk 1 above)
Comment (smathangi) [reply src/main/resources/application.yaml:99]: yes, thanks Matt. Fixed in 44d1567116b37283c5fc977c5fdc42a081aa428f

**Review by sullivangareth — DISMISSED** [review]

File: charts/ccd-case-document-am-api/values.preview.template.yaml [hunk 2]
```diff
@@ -3,4 +3,4 @@ java:
   ingressHost: ${SERVICE_FQDN}
   environment:
     DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal
```
Comment (mattnayler) [inline charts/ccd-case-document-am-api/values.preview.template.yaml:5]: Is`DEFINITION_STORE_HOST` still required if it has been remove from `src/main/resources/application.yaml`

**Review by mattnayler — DISMISSED** [review]
Approved. with one small comment/question.

File: charts/ccd-case-document-am-api/values.preview.template.yaml (hunk 2 above)
Comment (smathangi) [reply charts/ccd-case-document-am-api/values.preview.template.yaml]: Good catch Matt. Removed in 12f847262986e742103e153e9ae8480c676fab58

Comment (mattnayler) [reply charts/ccd-case-document-am-api/values.preview.template.yaml]: thanks update seen in 12f847262986e742103e153e9ae8480c676fab58

**Review by mattnayler — DISMISSED** [review]

**Pull request merged**